apps/docs/content/styled-system/concepts/html-elements.mdx

```diff
@@ -6,7 +6,7 @@ order: 3
 
 Hopper provides a set of HTML element components already configured with Hopper styled system. You should choose these components over native HTML elements.
 
-`<A>`, `<Address>`, `<Article>`, `<Aside>`, `<HtmlButton>`, `<Div>`, `<HtmlFooter>`, `<HtmlHeader>`, `<Img>`, `<Input>`, `<List>`, `<Main>`, `<Nav>`, `<Section>`, `<Span>`, `<Table>`
+`<A>`, `<Address>`, `<Article>`, `<Aside>`, `<HtmlButton>`, `<Div>`, `<HtmlFooter>`, `<HtmlHeader>`, `<Img>`, `<Input>`, `<List>`, `<Main>`, `<Nav>`, `<Section>`, `<Span>`, `<Table>`, `<THead>`, `<TBody>`, `<TFoot>`, `<TR>`, `<TD>`, `<TH>`, `<UL>`, `<OL>`, `<LI>`, `<Label>`, `<TextArea>`.
```

Comment:
Not you, but can you fix it at the same time.

it's not Section it's HtmlSection